sdk/samples/iot/README.md

```diff
@@ -24,8 +23,9 @@
     - [IoT Hub Telemetry Sample](#iot-hub-telemetry-sample)
     - [IoT Hub SAS Telemetry Sample](#iot-hub-sas-telemetry-sample)
     - [IoT Hub Twin Sample](#iot-hub-twin-sample)
-    - [IoT Hub Plug and Play Sample](#iot-hub-plug-and-play-sample)
-    - [IoT Hub Plug and Play Multiple Component Sample](#iot-hub-plug-and-play-multiple-component-sample)
+    - [IoT Plug and Play Sample](#iot-plug-and-play-sample)
+    - [IoT Plug and Play with Provisioning Sample](#iot-plug-and-play-with-provisioning-sample)
+    - [IoT Plug and Play Multiple Component Sample](#iot-plug-and-play-multiple-component-sample)
```

Comment:
```suggestion
    - [IoT Plug and Play Sample](#iot-plug-and-play-sample)
    - [IoT Plug and Play with Provisioning Sample](#iot-plug-and-play-with-provisioning-sample)
```

Reply:
The mulitple component is going to come be out best case end-of-day or worst-case very early next week.  So you're correct it's wrong for now, but it's going to get fixed ASAP and tree won't get merged till it is.  So I'm going to leave for now.